__tests__/util.test.ts

```diff
@@ -21,6 +21,14 @@ const HERE_RESPONSE_BUS =
   require('./json-mocks/here-response-bus.json') as FeatureCollection
 
 describe('arePointsEqual', () => {
+  it('should handle null inputs', () => {
+    // @ts-expect-error we are doing incorrect things
+    expect(arePointsRoughlyEqual([null, null], null)).toBe(false)
+    // @ts-expect-error we are doing incorrect things
+    expect(arePointsRoughlyEqual(null, [null, null])).toBe(false)
+    // @ts-expect-error we are doing incorrect things
```

Comment:
yeah....

Reply:
It's for a test, so this is supposed to happen